scripts/submit-analytics.js

```diff
@@ -122,6 +124,32 @@ function getRealmVersion() {
   return packageJson["version"];
 }
 
+/**
+ * Reads and parses `dependencies.list`.
+ * Each line has be form "KEY=VALUE", and we to find "REALM_CORE_VERSION"
+ *
+ * @returns the Realm Core version as a string
+ */
+function getRealmCoreVersion() {
+  const dependenciesListPath = path.resolve(__dirname, "../dependencies.list");
+  const dependenciesList = fse
+    .readFileSync(dependenciesListPath)
+    .toString()
+    .split("\n")
+    .map((s) => s.split("="));
+  return dependenciesList.find((e) => e[0] === "REALM_CORE_VERSION")[1];
+}
+
+/**
+ * Determines if `npm` or `yarn` is used.
+ *
+ * @returns An array with two elements: method and version
+ */
+function getInstallationMethod() {
+  const userAgent = process.env["npm_config_user_agent"];
+  return userAgent.split(" ")[0].split("/");
```

Comment:
Although I like the simplicity, I think it's a little brittle to depend on the order of these elements 🙂

Reply:
I think it conforms to [a typical user-agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent), and it will also have "product/version" in the beginning of the string.

Looking at the document for [npm](https://docs.npmjs.com/cli/v9/using-npm/config#user-agent) and [yarn](https://classic.yarnpkg.com/lang/en/docs/cli/config/) also support this, and I don't know if I can make it less brittle.